### PR TITLE
Add ability to install in a docker container.

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
ocd run https://ocd.appscale.com:8080/job/Daily%20Build/798/

File: debian/appscale_build.sh [hunk 1]
```diff
@@ -1,7 +1,31 @@
 #!/bin/bash
+#
+# This script install appscale on the local machine. It pulls in all the
```
Comment (cdonati) [inline debian/appscale_build.sh]: install -> installs. Also, probably AppScale for consistency with the next line.

File: debian/appscale_build.sh [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+# needed dependencies and configure them properly for AppScale.
```
Comment (cdonati) [inline debian/appscale_build.sh]: configure -> configures

File: debian/appscale_build.sh [hunk 3]
```diff
@@ -1,7 +1,31 @@
...
+# Some basic check: we need a way to install packages.
+PKG_CMD="$(which apt-get)"
+if [ -z "${PKG_CMD}" ]; then
+    echo "Cannot find the package manager command!"
+    exit 1
+fi
 
 set -e
 
+# Update the pakcages list and cache.
```
Comment (cdonati) [inline debian/appscale_build.sh]: packages

File: debian/appscale_build.sh [hunk 4]
```diff
@@ -1,7 +1,31 @@
...
+echo -n "Updating package list and cache ..."
+${PKG_CMD} update > /dev/null
+echo "done."
+
+# We need to make sure we have lsb-release, before we use it. On
+# streamlined images (like docker) it may not be present.
+if ! which lsb_release > /dev/null ; then 
+    echo -n "Installing lsb-release..."
+    (${PKG_CMD} install -y lsb-release) > /dev/null
```
Comment (cdonati) [inline debian/appscale_build.sh]: Is there a reason this is in a subshell while the previous PKG_CMD is not?

Comment (obino) [inline debian/appscale_build.sh]: not really, it seems cute at the time :) I had more commands there, so I used it for the redirection: the refactor brought it down to a single command: I will remove the subshell. Thanks!

**Pull request merged**